### PR TITLE
Report registration failure in ProcessTestCase

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: resolwe/test/testcases/process.py [hunk 1]
```diff
@@ -253,7 +253,10 @@ def setUp(self):
         """Initialize test data."""
         super().setUp()
 
-        self._register_schemas()
+        _, stderr = self._register_schemas()
+        stderr = stderr.getvalue()
+        if stderr:
+            raise ValueError(stderr)
```
Comment (dblenkus) [inline resolwe/test/testcases/process.py:259]: This is not `ValueError`, more `RuntimeError`. Otherwise, it looks good.

**Comment by dblenkus** [comment]
What about a change log?

File: resolwe/test/testcases/process.py (hunk 1 above)
Comment (JureZmrzlikar) [reply resolwe/test/testcases/process.py]: Fixed.

**Comment by JureZmrzlikar** [comment]
> What about a change log?

Just added. ;-0

**Comment by JureZmrzlikar** [comment]
Rebased, tests pass. Please merge.

**Pull request merged**